src/components/People/PersonCard.module.scss

```diff
@@ -37,13 +37,8 @@
 .personimage {
   position: absolute;
   bottom: 0;
-  right: 5%;
+  right: 0;
```

Comment:
This alone isn't enough to pull every image right, because object fit, width and maxheight also mess around with the location.

This set should do the trick

    position: absolute;
    bottom: 0;
    right: 0;
    max-width: 60%;
    max-height: 80%;